src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,43 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int INIT_POSITION = 0;
+    private static final int MAX_CAR_NAME_LENGTH = 5;
+    private final String name;
+
+    private int position;
```

Comment:
```suggestion
    private static final int MAX_CAR_NAME_LENGTH = 5;

    private final String name;
    private int position;
```
상수와 상태는 줄 바꿈을 통해서 가독성을 높여주세요! 😃